sycl/test-e2e/AddressSanitizer/out-of-bounds/local/group_local_memory.cpp

```diff
@@ -1,18 +1,17 @@
-// REQUIRES: linux, cpu
-// RUN: %{build} %device_sanitizer_flags -g -o %t.out
-// RUN: env SYCL_PREFER_UR=1 ONEAPI_DEVICE_SELECTOR=opencl:cpu %{run-unfiltered-devices} not %t.out 2>&1 | FileCheck %s
-
-#include <sycl/detail/core.hpp>
-
-#include <sycl/ext/oneapi/group_local_memory.hpp>
-#include <sycl/usm.hpp>
+// REQUIRES: linux
+// RUN: %{build} %device_sanitizer_flags -g -O0 -o %t.out
```

Comment:
Do we need to add tests to cover sycl external usage scenario?

Reply:
Currently we can't handle "sycl external" well because global variables are redefined in libdevice.
I will fix this when we support link libdevice in UR.